bin/dev

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+# Default to port 3000 if not specified
+export PORT="${PORT:-3000}"
```

Comment:
Should we set a default `RAILS_ENV=development` here?

Reply:
We could. I just copy/pasted this from the rails-generated default in another branch. I think because RAILS_ENV defaults to development you don't usually need it, but because we have that boot.rb env var check thing, we might.

Reply:
Yes, this was my intent, to make it as easy as possible to get started. I do not see when someone might want to launch `bin/dev` without `RAILS_ENV=development`

Reply:
Pushed commit which includes dev env default